pontoon/static/js/utils.js

```diff
@@ -0,0 +1,149 @@
+
+export class Cookies {
+
+    get (name) {
+        var cookie = document.cookie.match('(^|;)\\s*' + name + '\\s*=\\s*([^;]+)');
+        return cookie ? cookie.pop() : '';
+    }
+}
+
+
+export class Strip {
+
+    strip (str, char) {
+        return this.rstrip(this.lstrip(str, char), char)
+    }
+
+    lstrip (str, char) {
+        return str.replace(new RegExp('^' + (char || '\\s') + '+'), '');
+    }
+
+    rstrip (str, char) {
+        const match = new RegExp('^[^' + (char || '\\s') + ']')
+        for (var i = str.length - 1; i >= 0; i--) {
+            if (match.test(str.charAt(i))) {
+                return str.substring(0, i + 1);
+            }
+        }
+    }
+}
+
+
+export class DjangoAjax {
+
+    get csrf () {
+        return cookies.get('csrf');
+    }
+
+    get headers () {
+        return new Headers({
+            'X-Requested-With': 'XMLHttpRequest',
+            'X-CSRFToken': this.csrf,
+            "Accept": "application/json"});
+    }
+
+    asGetParams (data) {
+        return this.appendParams(new URLSearchParameters(), data);
+    }
+
+    asMultipartForm (data) {
+        return this.appendParams(new FormData(), data);
+    }
+
+    fetch (url, data, params) {
+        const {method} = (params || {})
```

Comment:
`const {method} = (params || {'method': 'get'})` because currently if you won't pass params, function will return null.
Also I'd raise  an exception if passed method is unsupported.

Reply:
cool, ill add